bin/_util.js

```diff
@@ -140,4 +140,114 @@ module.exports = {
       }
     }
   },
+  /**
```

Comment:
Nit: I'd be tempted to put the utils under `src/` and point the `edu-import-from-figma` script at `build/`, so that I could use TS for these.

Reply:
I don't mind that. While working on these scripts, I have run into a few ideas to improve them, and will add a note for this as well. Some includes updating the script(s) to be TS, and consolidating them such that they are subcommands instead of individual files (e.g., `npx eds-theme {init|apply|import}`, among other refactors.

One thing I was hoping to have (or make, if it doesn't exist) is types for the figma data, so that any tools around that can be safer.